client/sites-dashboard/components/sites-grid-item.tsx

```diff
@@ -160,12 +160,14 @@ export const SitesGridItem = memo( ( props: SitesGridItemProps ) => {
 						<div className={ badges }>
 							{ isP2Site && <SitesP2Badge>P2</SitesP2Badge> }
 							{ isWpcomStagingSite && <SitesStagingBadge>{ __( 'Staging' ) }</SitesStagingBadge> }
-							{ isMigrationTrialPlanSite && (
+							{ ( isMigrationTrialPlanSite || isECommerceTrialSite ) && (
```

Comment:
Instead of having these two checks here, could we leveraging the existing `isTrialSite` selector [here](https://github.com/Automattic/wp-calypso/blob/73ed67a36036518f8aa8ad6fa10bf710d18b4911/client/state/sites/plans/selectors/trials/is-trial-site.ts)?

Reply:
Thanks for pointing to the selector. TBH, I didn't know that isTrialSite select exists.

I'll update it accordingly!